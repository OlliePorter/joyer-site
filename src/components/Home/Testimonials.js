import React from 'react';
import Typography from '@material-ui/core/Typography';
import Grid from '@material-ui/core/Grid';
import { makeStyles } from '@material-ui/core/styles';
import theme from '../../theme';

const useStyles = makeStyles({
  coverImage: {
    backgroundImage: 'url(/smiling.jpg)',
    backgroundSize: 'cover',
    backgroundRepeat: 'no-repeat',
    backgroundPosition: 'center',
    minHeight: '45vh',
  },
  coverImageOverlay: {
    backgroundColor: 'rgba(0, 0, 0, 0.5)',
    position: 'absolute',
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
    justifyContent: 'center',
    minHeight: '45vh',
    width: '100%',
    overflowX: 'hidden',
  },
  testimonialContainer: {
    // backgroundColor: 'red',
  },
  testimonialText: {
    color: 'white',
    paddingLeft: '120px',
    paddingRight: '120px',
    textAlign: 'left',
    fontWeight: 300,
  },
  testimonialImage: {
    width: '60px',
    height: '60px',
    borderRadius: '50%',
  }
});

export default function Testimonials() {
  const classes = useStyles();

  return(
    <>
      <div className={classes.coverImage}>
        <div className={classes.coverImageOverlay}>
          <Grid container spacing={10}>
            <Grid item xs={4}>
              <div className={classes.testimonialContainer}>
                <Typography className={classes.testimonialText} variant="h6">
                  "I feel so much closer to my friends after downloading this app. Go JOYER!"
                </Typography>
                <img src={'/lucy_f.jpg'} alt="Female Image" className={classes.testimonialImage}/>
              </div>
            </Grid>
            <Grid item xs={4}>
              <div className={classes.testimonialContainer}>
                <Typography className={classes.testimonialText} variant="h6">
                  "I love being able to set reminders to catch up with my network."
                </Typography>
              </div>
            </Grid>
            <Grid item xs={4}>
              <div className={classes.testimonialContainer}>
                <Typography className={classes.testimonialText} variant="h6">
                  "Blind get-togethers is such a good feature! A must download."
                </Typography>
              </div>
            </Grid>
          </Grid>
        </div>
      </div>
    </>
  );
}
