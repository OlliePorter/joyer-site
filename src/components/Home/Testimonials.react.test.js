import React from 'react';
import Testimonials from './Testimonials';
import renderer from 'react-test-renderer';

it('Testimonials renders correctly', () => {
    const tree = renderer.create(
      <Testimonials />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });