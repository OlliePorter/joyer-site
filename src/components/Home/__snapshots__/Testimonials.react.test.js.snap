// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testimonials renders correctly 1`] = `
<div
  className="makeStyles-coverImage-1"
>
  <div
    className="makeStyles-coverImageOverlay-2"
  >
    <div
      className="makeStyles-coverPadding-3"
    >
      <div
        className="MuiGrid-root makeStyles-testimonialGrid-11 MuiGrid-container MuiGrid-spacing-xs-8"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4"
        >
          <h6
            className="MuiTypography-root makeStyles-testimonialText-6 MuiTypography-h6"
          >
            "I feel so much closer to my friends after downloading this app. Go JOYER!"
          </h6>
          <div
            className="makeStyles-testimonialRowContainer-4"
          >
            <img
              alt="Female Image"
              className="makeStyles-testimonialImage-9"
              src="/lucy_f.jpg"
            />
            <div
              className="makeStyles-testimonialColumnContainer-5"
            >
              <p
                className="MuiTypography-root makeStyles-testimonialTextTitle-7 MuiTypography-body1"
              >
                Lucy F
              </p>
              <p
                className="MuiTypography-root makeStyles-testimonialTextSubtitle-8 MuiTypography-body1"
              >
                Early Joyager
              </p>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4"
        >
          <h6
            className="MuiTypography-root makeStyles-testimonialText-6 MuiTypography-h6"
          >
            "I love being able to set reminders to catch up with my network."
          </h6>
          <div
            className="makeStyles-testimonialRowContainer-4"
          >
            <img
              alt="Female Image"
              className="makeStyles-testimonialImage-9"
              src="/frederick_s.jpg"
            />
            <div
              className="makeStyles-testimonialColumnContainer-5"
            >
              <p
                className="MuiTypography-root makeStyles-testimonialTextTitle-7 MuiTypography-body1"
              >
                Frederick S
              </p>
              <p
                className="MuiTypography-root makeStyles-testimonialTextSubtitle-8 MuiTypography-body1"
              >
                Early Joyager
              </p>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4"
        >
          <h6
            className="MuiTypography-root makeStyles-testimonialText-6 MuiTypography-h6"
          >
            "Joy Coins are great! Hands down the best way to earn money off food and drinks."
          </h6>
          <div
            className="makeStyles-testimonialRowContainer-4"
          >
            <img
              alt="Female Image"
              className="makeStyles-testimonialImage-9"
              src="/gabbie_h.jpg"
            />
            <div
              className="makeStyles-testimonialColumnContainer-5"
            >
              <p
                className="MuiTypography-root makeStyles-testimonialTextTitle-7 MuiTypography-body1"
              >
                Gabbie H
              </p>
              <p
                className="MuiTypography-root makeStyles-testimonialTextSubtitle-8 MuiTypography-body1"
              >
                Early Joyager
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Testimonials renders correctly 1`] = `
<div
  className="makeStyles-coverImage-1"
>
  <div
    className="makeStyles-coverImageOverlay-2"
  >
    <div
      className="makeStyles-coverPadding-3"
    >
      <div
        className="MuiGrid-root makeStyles-testimonialGrid-11 MuiGrid-container MuiGrid-spacing-xs-8"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4"
        >
          <h6
            className="MuiTypography-root makeStyles-testimonialText-6 MuiTypography-h6"
          >
            "I feel so much closer to my friends after downloading this app. Go JOYER!"
          </h6>
          <div
            className="makeStyles-testimonialRowContainer-4"
          >
            <img
              alt="Female Image"
              className="makeStyles-testimonialImage-9"
              src="/lucy_f.jpg"
            />
            <div
              className="makeStyles-testimonialColumnContainer-5"
            >
              <p
                className="MuiTypography-root makeStyles-testimonialTextTitle-7 MuiTypography-body1"
              >
                Lucy F
              </p>
              <p
                className="MuiTypography-root makeStyles-testimonialTextSubtitle-8 MuiTypography-body1"
              >
                Early Joyager
              </p>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4"
        >
          <h6
            className="MuiTypography-root makeStyles-testimonialText-6 MuiTypography-h6"
          >
            "I love being able to set reminders to catch up with my network."
          </h6>
          <div
            className="makeStyles-testimonialRowContainer-4"
          >
            <img
              alt="Female Image"
              className="makeStyles-testimonialImage-9"
              src="/frederick_s.jpg"
            />
            <div
              className="makeStyles-testimonialColumnContainer-5"
            >
              <p
                className="MuiTypography-root makeStyles-testimonialTextTitle-7 MuiTypography-body1"
              >
                Frederick S
              </p>
              <p
                className="MuiTypography-root makeStyles-testimonialTextSubtitle-8 MuiTypography-body1"
              >
                Early Joyager
              </p>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4"
        >
          <h6
            className="MuiTypography-root makeStyles-testimonialText-6 MuiTypography-h6"
          >
            "Joy Coins are great! Hands down the best way to earn money off food and drinks."
          </h6>
          <div
            className="makeStyles-testimonialRowContainer-4"
          >
            <img
              alt="Female Image"
              className="makeStyles-testimonialImage-9"
              src="/gabbie_h.jpg"
            />
            <div
              className="makeStyles-testimonialColumnContainer-5"
            >
              <p
                className="MuiTypography-root makeStyles-testimonialTextTitle-7 MuiTypography-body1"
              >
                Gabbie H
              </p>
              <p
                className="MuiTypography-root makeStyles-testimonialTextSubtitle-8 MuiTypography-body1"
              >
                Early Joyager
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
